### ISSUE TITLE
BrowserDomAdapter: Uncaught UnsubscriptionError - Rx subscription issue

### ISSUE THREAD

**Issue by MarkPieszak** [angular/universal#350]
Made sure to get latest just this morning for the *universal-starter*. 

Same issue as #343 

Once the project gets up and running with `npm start` there seems to be errors coming up for `BrowserDomAdapter`. Trying to figure out how this is related to Universal (or whether it's an Angular2 issue itself).

It seems to all be tied into:

```
// Our API for demos only
app.get('/data.json', (req, res) => {
  res.json({
    data: 'fake data'
  });
});
```

As well as `app.component.ts` here: **Also: commenting out this line makes the issue go away.**

![screen shot 2016-04-06 at 12 32 21 pm](https://cloud.githubusercontent.com/assets/2574412/14324661/ac6c4d54-fbf3-11e5-8272-ad05266e529f.png)

![screen shot 2016-04-06 at 11 58 17 am](https://cloud.githubusercontent.com/assets/2574412/14323518/e2275e8e-fbee-11e5-9ac9-0ec291e8c4bf.png)

-----
One of errors when going line by line creeps up here, and you can see it's from right after the http request being made.

![screen shot 2016-04-06 at 12 14 47 pm](https://cloud.githubusercontent.com/assets/2574412/14324075/5ae4c652-fbf1-11e5-859f-12469b9b1f93.png)

* **I'm submitting a ...**
- [X] bug report

* **What modules are related to this pull-request**
- [X] universal

Mac OS El Capitan & Chrome

**Comment by MarkPieszak** [comment]
Thanks to Mega in slack it looks like this is a **Zone** issue itself not on us.
Reverting down to `0.6.8` fixes this issue.

Zone issue here : https://github.com/angular/zone.js/issues/310

**Comment by MarkPieszak** [comment]
Closed due to `0.6.10` being added to project.
